subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -77,18 +77,14 @@ The related APIs in `StartParameter` are now deprecated.
 These methods currently do not work as expected.
 They are being deprecated to avoid confusion.
 
-=== Potential breaking changes
-
-==== Android Gradle Plugin 3.3 and earlier is not supported anymore
-
-Gradle 6.0 supports Android Gradle Plugin versions 3.4 and later.
+==== Copy or archive duplicate paths without explicitly specifying a duplicates strategy has been deprecated
 
-==== Archive tasks fail on duplicate files
+Copy and archive tasks default to the `INCLUDE` duplicates strategy, allowing the same path to be copied multiple times to the destination or to exist multiple times in an archive.
```

Comment:
```suggestion
Archive tasks `Tar` and `Zip` by default allow multiple entries for the same path to exist in the created archive.
These https://github.com/gradle/gradle/issues/9990["grossly invalid zip files" can trigger zip bomb detection].
To prevent this from happening accidentally encountering duplicates while creating an archive now produces a deprecation message, and will fail the build starting with Gradle 7.0.

Note that copy tasks also happily copy multiple sources with the same relative path to the destination directory.
That behavior has also been deprecated.
```

Reply:
Done.